src/emscripten-optimizer/simple_ast.h

```diff
@@ -396,19 +396,19 @@ struct Value {
         }
         os << '[';
         if (pretty) {
-          os << std::endl;
```

Comment:
Yeah I've seen projects where `std::endl` is banned in favor of just '\n' because it gets expanded anyway and they didn't want the extra flush.